api/models/Post.js

```diff
@@ -163,6 +163,14 @@ module.exports = bookshelf.Model.extend(Object.assign({
     return this.hasMany(ProjectContribution)
   },
 
+  proposalOptions: function () {
+    return this.hasMany(ProposalOption)
```

Comment:
for performance purposes you can do     return this.get('type') === Post.Type.PROPOSAL ? this.hasMany(ProposalOption) : null
This means that if the front end loads proposalOptions for a stream of posts then it won't do the query to try and load them for non proposal posts, saving a ton of queries.